src/main/kotlin/no/nav/syfo/dialogmote/domain/Dialogmote.kt

```diff
@@ -31,10 +32,11 @@ fun Dialogmote.toDialogmoteDTO() =
         tildeltEnhet = this.tildeltEnhet,
         arbeidstaker = this.arbeidstaker.toDialogmotedeltakerArbeidstakerDTO(),
         arbeidsgiver = this.arbeidsgiver.toDialogmotedeltakerArbeidsgiverDTO(),
-        tidStedList = this.tidStedList.map {
-            it.toDialogmoteTidStedDTO()
-        },
+        sted = dialogmoteTidSted?.sted.orEmpty(),
+        tid = dialogmoteTidSted?.tid ?: LocalDateTime.MIN,
```

Comment:
Kan "dialogmoteTidSted" være nullable? Er ikke det påkrevd for å få opprettet et møte?

Om ikke kan vi jo droppe "?" og default verdier. Men om de kan være null, så tror jeg kanskje det er bedre å ikke defaulte til LocalDateTime.MIN, da frontend da vil se at der finnes en verdi og prøve og vise den. Dersom den er null så kan vi unngå å vise den.

Reply:
Ja, denne skal i praksis aldri bli null, så denne kan endres til:
`tid = dialogmoteTidSted!!.tid` evt. ?: null  som Audun nevner om man vil "ignorere" manglende verdi.

Reply:
Fikset nå